f600233d1b154d507b782f84604bcb12/update/sys_script_include_38d1130197141110c351f5b3f153af83.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_script_include">
+    <sys_script_include action="INSERT_OR_UPDATE">
+        <access>package_private</access>
+        <active>true</active>
+        <api_name>x_g_dis_atat.edaPdsRequest</api_name>
+        <caller_access/>
+        <client_callable>true</client_callable>
+        <description>Makes a GET to the EDA PDS Test Endpoint using mTLS.&#13;
+&#13;
+When invoking, pass a payload object:&#13;
+&#13;
+payload: {&#13;
+endpoint: string (required),&#13;
+contract_number: string (optional),&#13;
+delivery_order_number: string (optional)&#13;
+}&#13;
+</description>
+        <name>edaPdsRequest</name>
+        <script><![CDATA[function edaPdsRequest(payload){
+	try{
+		// outbound REST message
+		var request = new sn_ws.RESTMessageV2();
+		
+		// request set-up
+		request.setEndpoint(payload.endpoint);
+		request.setHttpMethod("get");
+		request.setLogLevel("all");
+		request.setMutualAuth("mauth");
+		
+		// add query parameters
+		if(payload.contract_number){
+			request.setQueryParameter("contractNumber", payload.contract_number);
+		}
+		if(payload.delivery_order_number){
+			request.setQueryParameter("deliveryOrderNumber", payload.delivery_order_number);
+		}
+		
+		// make request
+		var response = request.execute();
+		// check for errors
+		if(response.getStatusCode() != "200" && response.getStatusCode() != "204"){
```

Comment:
201 needed here?

Reply:
I wasn't able to trigger a `201` response with the `GET`, since it isn't creating any content.
Sidenote:
The API is a bit weird because it would return a `204 No Content` if the **contractNumber** was wrong:
![image](https://user-images.githubusercontent.com/84199040/175574669-87db643d-7611-45d9-9263-fefb8afad2e8.png)

But it would give a `400` if the **deliveryOrderNumber** was wrong:
![image](https://user-images.githubusercontent.com/84199040/175574738-7b4807d3-a4ab-4a2a-904c-a36efcf8638b.png)
Weird.

Reply:
OK just checking